tomni/cytosmart_data_format/annotations/ellipse/main.py

```diff
@@ -1,42 +1,133 @@
-from cell_fea.morphology import circularity
+from dataclasses import asdict
+from typing import List
+
+import numpy as np
 
 from ..annotation import Annotation
 from ..point import Point
 
 
 class Ellipse(Annotation):
-    def __init__(self, radius: Point, center: Point, rotation: float):
+    def __init__(
+        self,
+        radius: Point,
+        center: Point,
+        rotation: float,
+        id: str,
+        label: str,
+        children: List[Annotation],
+        parents: List[Annotation],
+    ):
```

Comment:
add docstring

Reply:
Mainly make sure the units are known. Is rotation is fraction, radii, or degrees